### ISSUE TITLE
org-mode File mode specification error: (json-readtable-error 122)

### ISSUE THREAD

**Issue by erikcw** [syl20bnr/spacemacs#13206]
#### Description :octocat:
Opening any `.org` file results in `File mode specification error: (json-readtable-error 122)`.

#### Reproduction guide :beetle:
- Start Emacs
- Open .org file (can be blank).

In `ielm` this can be reproduced with:

```
ELISP> (with-temp-buffer (org-mode))
*** Eval error ***  JSON readtable error: 122
```

*Observed behaviour:* :eyes: :broken_heart:
`File mode specification error: (json-readtable-error 122)`

*Expected behaviour:* :heart: :smile:
No errors :smile:.

#### System Info :computer:
- OS: darwin
- Emacs: 26.3
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 96e7621f8)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(lsp dap go sql csv nginx markdown ruby helm auto-completion emacs-lisp osx git docker ansible markdown multiple-cursors treemacs org
     (shell :variables shell-default-height 30 shell-default-position 'bottom)
     spell-checking version-control lua
     (python :variables python-backend 'lsp)
     ipython-notebook django clojure javascript react html yaml php terraform asciidoc
     (ranger :variables ranger-show-preview t)
     pass notmuch
     (rcirc :variables rcirc-enable-authinfo-support t))
```
- System configuration features: JPEG RSVG IMAGEMAGICK GLIB NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS LCMS2

#### Backtrace :paw_prints:
```
File mode specification error: (json-readtable-error 122)

```

**Comment by duianto** [comment]
The same error message is mentioned in this issue:
org-capture throws json-readtable-error #9941

The `ipython-notebook` layer is mentioned as the source.

When I added the `ipython-notebook` layer in my PopOS 19.10 Virtualbox VM, 
and opened a `.org` file, then I got this error message:
>File mode specification error: (json-readtable-error 47)

and in `ielm`:
```
*** Welcome to IELM ***  Type (describe-mode) for help.
ELISP> (with-temp-buffer (org-mode))
*** Eval error ***  JSON readtable error: 47
```

A comment in that issue https://github.com/syl20bnr/spacemacs/issues/9941#issuecomment-546484753, mentions that't it's fixed by installing `jupyter`:
```
pip install jupyter
```

And when I did that and added it's install path:
`/home/username/.local/bin`
to the end of my `.bashrc`
```
export PATH=$PATH:/home/username/.local/bin
```
And restarted Spacemacs.

Now there's no error message when i open a `.org` file.
And the command in `ielm` looks like this:
```
*** Welcome to IELM ***  Type (describe-mode) for help.
ELISP> (with-temp-buffer (org-mode))
nil
```
